ballerina/jwt_issuer.bal

```diff
@@ -81,6 +81,8 @@ public isolated function issue(IssuerConfig issuerConfig) returns string|Error {
     var config = signatureConfig?.config;
     if (config is ()) {
         return prepareError("Signing JWT requires keystore information or private key information.");
+    } else if (config is string) {
```

Comment:
```suggestion
    } else if config is string {
```
Check for other places as well

Reply:
@ayeshLK Thanks for pointing out these. I will make sure to update the complete codebase along with https://github.com/ballerina-platform/ballerina-standard-library/issues/1718. If I update only here, the existing codebase will be inconsistent or if I update the whole code base with this, the objective of this PR will go away.